.gitmodules

```diff
@@ -9,7 +9,7 @@
 	url = https://github.com/ambrosiogabe/CppUtils
 [submodule "Animations/vendor/ffmpeg"]
 	path = Animations/vendor/ffmpeg
-	url = https://git.ffmpeg.org/ffmpeg.git
+	url = https://github.com/FFmpeg/FFmpeg.git
```

Comment:
The official FFMPEG server is acting up, changing to the Github mirror fixes these issues.